metadata-utils/src/main/java/com/linkedin/metadata/utils/SearchUtil.java

```diff
@@ -17,14 +18,18 @@ public class SearchUtil {
   private SearchUtil() {
   }
 
-  public static List<FilterValue> convertToFilters(Map<String, Long> aggregations) {
+  /*
+   * @param aggregations the aggregations coming back from elasticsearch combined with the filters from the search request
+   * @param filteredValues the set of values provided by the search request
+   */
+  public static List<FilterValue> convertToFilters(Map<String, Long> aggregations, Set<String> filteredValues) {
```

Comment:
Nice. Do we have any unit tests for this SearchUtil test?

Reply:
added